### ISSUE TITLE
Issue with configuring automatic updates

### ISSUE THREAD

**Issue by lighthouse64** [iSpec/Hyperdrive#1]
sudo dpkg-reconfigure -plow unattended-upgrades gives me an error when I run it.

It says that the start and stop actions are no longer supported

**Comment by iSpec** [comment]
Probably not a problem with the code. This should work on a fresh ubuntu image, so issue is resolved.

**Comment by lighthouse64** [comment]
nope.  I just installed a new ubuntu image, and it didn't work.  :(
